Sources/TuistKit/Commands/Dependencies/DependenciesFetchCommand.swift

```diff
@@ -0,0 +1,22 @@
+import ArgumentParser
+import Foundation
+import TSCBasic
+
+/// A coomand to fetch project's dependencies.
+struct DependenciesFetchCommand: ParsableCommand {
+    static var configuration: CommandConfiguration {
+        CommandConfiguration(commandName: "fetch",
+                             abstract: "Fetches project's dependecies ")
```

Comment:
```suggestion
                             abstract: "Fetches project's dependencies defined in Dependencies.swift ")
```

Reply:
And I'd say: `Fetches the project's....`